swagger_server/controllers/tree_get_controller.py

```diff
@@ -0,0 +1,25 @@
+from swagger_server.db import get_db
+from swagger_server.exceptions import NotFound
+from swagger_server.models.error import Error  # noqa: E501
+from swagger_server.ogm.leaf_node import LeafNode
+
+
+def tree_id_get(id):  # noqa: E501
```

Comment:
The original idea was to get the list of samples that has nearest leaf node based on an ID. But after another thought I also think it is better to return the leaf itself for this endpoint. I have updated the spec in this change da1db82.

I will add another endpoint to get the list of samples that have the same nearest leaf in another pull request.